### DISCUSSION TITLE
Can Mountain Duck Establish A Connection At Login?

### DISCUSSION THREAD

**Discussion by WhatImListnTo** [iterate-ch/cyberduck#16070] [Q&A] [answered]
I looked through preferences, but I did not find an option to establish a connection at login.  Is this possible via command-line? (i.e., can I add this to registry [HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Run])

If the answer to this is no, I will post this as a feature request.  I have an SFTP server that contains data I always use and wish connected whenever I am logged in.  This may be useful to others as well.

**Comment by tatze96** [comment]
Automatic bookmark reconnects at login only work if the "Save Workspace" option within the Mountain Duck preferences _General_ tab and the "Enable Login Item" option within the _Login Item_ tab are activated and the bookmark was mounted before quitting Mountain Duck.

**Comment by WhatImListnTo** [comment]
Thank you for responding.  I actually have both of these options enabled now.  I have been disconnecting prior to shutdown to prevent any data loss.  This explains why it starts up not connected for me.

**Comment by WhatImListnTo** [comment]
I would like to see a command-line parameter added to Mountain Duck to establish a previously defined bookmark connection.  This way I can create a shortcut with this command-line parameter to have a connection established via HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run

I do not feel safe shutting down my machine with synchronization still open and running.  This is the reason I disconnect prior to shut down.

**Reply by dkocher** [reply, marked as answer]
You may find [Connect Using the Command Line Interface](https://docs.mountainduck.io/mountainduck/interface/#connect-using-the-command-line-interface) helpful.

**Reply by WhatImListnTo** [reply]
Thanks for posting this.  I had looked all over the documentation and could not find this earlier.

This will solve my issues.
